src/plugins/data_source_management/public/management_app/mount_management_section.tsx

```diff
@@ -45,7 +55,8 @@ export async function mountManagementSection(
               <CreateDataSourceWizardWithRouter />
             </Route>
             <Route path={['/']}>
-              <DataSourceTableWithRouter />
+              <DataSourceTable 
+                setBreadcrumbs={deps.setBreadcrumbs} />
```

Comment:
setBreadcrumbs is already part of the services on 50, how about we use from there directly instead of import it in each page?

Reply:
done